samples/test/stitcher.test.js

```diff
@@ -84,12 +84,16 @@ before(() => {
     .forEach(line => {
       if (line.includes(`locations/${location}/slates/${slateIdPrefix}`)) {
         this.nextId = line.split('/').pop();
-        execSync(
-          `node deleteSlate.js ${projectId} ${location} ${this.nextId}`,
-          {
-            cwd,
-          }
-        );
+        try {
+          execSync(
+            `node deleteSlate.js ${projectId} ${location} ${this.nextId}`,
+            {
+              cwd,
+            }
+          );
+        } catch (err) {
```

Comment:
Do we need to check that the err is the NotFound error?

Reply:
Plus one, we should make sure we're not catching all errors

Reply:
Refactored to ignore NOT_FOUND